newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/EventsFunctionParametersEditor.js

```diff
@@ -292,29 +293,48 @@ export default class EventsFunctionParametersEditor extends React.Component<
       );
     }
 
+    const parameters =
+      eventsFunctionsContainer &&
+      eventsFunction.getFunctionType() === gd.EventsFunction.ActionWithOperator
+        ? eventsFunction.getParametersForEvents(eventsFunctionsContainer)
+        : eventsFunction.getParameters();
+
+    const firstParameterIndex = eventsBasedBehavior
+      ? 2
+      : eventsBasedObject
+      ? 1
+      : 0;
     const isParameterDisabled = index => {
       return (
-        !!freezeParameters ||
-        (!!eventsBasedBehavior && index < 2) ||
-        (!!eventsBasedObject && index < 1)
+        eventsFunction.getFunctionType() ===
+          gd.EventsFunction.ActionWithOperator ||
+        freezeParameters ||
+        index < firstParameterIndex
       );
     };
-    const isParameterDescriptionAndTypeShown = index => {
+    const typeShownFirstIndex = firstParameterIndex;
+    const labelShownFirstIndex =
+      firstParameterIndex +
+      (eventsFunction.getFunctionType() === gd.EventsFunction.ActionWithOperator
+        ? 1
+        : 0);
+    const isParameterTypeShown = index => {
       // The first two parameters of a behavior method should not be changed at all,
       // so we even hide their description and type to avoid cluttering the interface.
       // Same thing for an object which has mandatory Object parameter.
```

Comment:
Move this comment before the declaration of "typeShownFirstIndex" (because that's where the logic is now)